### PR TITLE
Allow ScorllViewer to manage PointerWheel Event

### PR DESCRIPTION
**Pull request by workgroupengineering**
## Checklist

- [ ] Added unit tests (if possible)?
- [x] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

### CODE REVIEW

**Comment by kekekeks** [comment]
Is there a similar setting in WPF/UWP? If there is we should prefer to use the same API.

**Comment by grokys** [comment]
Could we get a description on this? I think I can see what it does, but could you explain:

- Why one would need this?
- Whether its present on other frameworks?
- Why no unit tests?

You may have noticed PRs by the core team being opened and merged without good descriptions; that's less than ideal but happens because it's something we have discussed in chat so everyone is on the same page. For PRs from non-core contributors, especially PRs that haven't been talked about before in chat, a good description will help enormously to get a prompt review (btw our contributing guidelines [do mention](https://github.com/AvaloniaUI/Avalonia/blob/master/CONTRIBUTING.md#before-you-start) that it's best to chat with us to about a feature/fix before starting work).

**Comment by workgroupengineering** [comment]
> 
> 
> Is there a similar setting in WPF/UWP? If there is we should prefer to use the same API.

There is no equivalent in PointerWheelScrollMode in WPF

**Comment by workgroupengineering** [comment]
>     * Why one would need this?

Because it is faster to scroll with the wheel on the desktop than to move the mouse

>     * Whether its present on other frameworks?

WPF



>     * Why no unit tests?

Because there are already Unit Tests that check Offset and this functionality is closely linked to it. If you want I add a test.



> You may have noticed PRs by the core team being opened and merged without good descriptions; that's less than ideal but happens because it's something we have discussed in chat so everyone is on the same page. For PRs from non-core contributors, especially PRs that haven't been talked about before in chat, a good description will help enormously to get a prompt review (btw our contributing guidelines [do mention](https://github.com/AvaloniaUI/Avalonia/blob/master/CONTRIBUTING.md#before-you-start) that it's best to chat with us to about a feature/fix before starting work).

Sure, but in any case I would have implemented it because I need it, so I have to share it.

File: src/Avalonia.Controls/ScrollViewer.cs [hunk 1]
```diff
@@ -713,5 +714,44 @@ private void RaiseScrollChanged()
...
+                if (_pointerWheelScrollMode != value)
+                {
+                    var old = _pointerWheelScrollMode;
+                    _pointerWheelScrollMode = value;
+                    RaisePropertyChanged(PointerWheelScrollModeProperty, old, value);
+                }
+            }
+        }
+
+        protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
```
Comment (kekekeks) [inline src/Avalonia.Controls/ScrollViewer.cs:744]: The event is handled in https://github.com/AvaloniaUI/Avalonia/blob/fec8341176acf383233e218422c613ad9fc3adbf/src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs#L396

Why do we need a separate handler here?

Comment (workgroupengineering) [reply src/Avalonia.Controls/ScrollViewer.cs:744]: It is my mistake. I Close PR.

**Pull request closed without merging**